test/e2e-test/profile/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.12.1-alpine3.19
+
+ARG USER=ccnp
+ARG UID=1000
+ARG GID=1000
+ARG GROUP=ccnp
+
+WORKDIR /run/confidential-cloud-native-primitives
+COPY  ./ ./
+RUN addgroup -S -g $GID $GROUP && adduser -S -u $UID -D -G $GROUP $USER
+RUN chown -R  $USER:$GROUP  ./
+
+RUN apk update && apk add bash vim
+RUN pip install -r sdk/python3/requirements.txt && pip install pytest pytdxattest
+
+USER $UID
+
+COPY --from=golang:1.20-alpine /usr/local/go/ /usr/local/go/
+ENV PATH="/usr/local/go/bin:${PATH}"
+ENV GOROOT="/usr/local/go"
+ENV GOPATH="/home/ccnp/gopath"
+
+RUN pip install  ./sdk/python3/
+RUN cd ./test/e2e-test/go-test && go mod tidy && cd ../../..
+
+ENTRYPOINT ["tail", "-f", "/dev/null"]
```

Comment:
A little bit strange ENTRYPOINT, may I know the reason?

Reply:
This ENTRYPOINT will keep docker container running and wait for subsequent commands.
The ci-e2e-test.sh script will enter this container to run multiple test cases.